packages/www/pages/app/admin/user/[id].tsx

```diff
@@ -51,7 +51,8 @@ const ID = () => {
   const [enableModal, setEnableModal] = useState(false);
   const [updateViewerLimitModal, setUpdateViewerLimitModal] = useState(false);
   const [newViewerLimit, setNewViewerLimit] = useState(
-    userInfo?.viewerLimit || 50000,
+    userInfo?.viewerLimit ||
+      (userInfo?.stripeProductId === "prod_O9XuIjn7EqYRVW" ? 30 : 10_000),
```

Comment:
What is this `prod_O9XuIjn7EqYRVW`

Reply:
It's the sandbox plan. Unluckily I have to hardcode it until we have products in a proper `product` table when we finally do billing v2